src/commands/lambda/functions/commons.ts

```diff
@@ -180,11 +180,8 @@ export const isMissingAWSCredentials = () =>
 
 export const isMissingDatadogSiteEnvVar = () => {
   const site = process.env[CI_SITE_ENV_VAR]
-  if (site !== undefined) {
-    return !isValidDatadogSite(site)
-  }
 
-  return true
+  return site === undefined || !isValidDatadogSite(site)
```

Comment:
The function `isValidDatadogSite` accepts a parameter `site` of type `string`, I could change by doing the following:

```ts
export const isValidDatadogSite = (site?: string): boolean => {
  if (site === undefined) {
    return false
  }

  return !!process.env.DD_CI_BYPASS_SITE_VALIDATION || DATADOG_SITES.includes(site.toLowerCase())
}
```

Allowing the removal of the `isMissing...` method. 
I think this might trigger an approval from an external just for that method.

What do you think of this approach?

Reply:
Updated the function in `helpers/validation.ts` and added documentation.